airbrakes/airbrakes.py

```diff
@@ -91,15 +91,15 @@ def update(self) -> None:
         # data packet
         i = 0
         for data_packet in data_packets:
-            logged_data_packet = LoggedDataPacket(
+            ldp = LoggedDataPacket(
```

Comment:
I don't think acronyms are every good var names, it makes it less clear

Reply:
and this isn't just me, I read this dude's book and it's really good. I highly recommend. https://www.google.com/books/edition/Clean_Code/_i6bDeoCQzsC?hl=en&gbpv=1&printsec=frontcover